### DISCUSSION TITLE
Change theme-color content

### DISCUSSION THREAD

**Discussion by alex0xhodler** [AzuraCast/AzuraCast#5831] [Q&A]
Hey everyone, managed to customize everything in the page using css and tags, but I was not able to change the following line from <head> 

<meta name="theme-color" content="#2196F3">

Anyone has any idea how can I change this easily for the public facing pages? 

Thanks,
Alex

**Comment by krono-i2** [comment]
You can use javascript for modifying the tag
```
<meta name="theme-color" content="#2196F3">
```
For example:
```
document.getElementsByName('theme-color')[0].setAttribute('content','#FFFFFF');
```
